sparql/sparql11/protocol/manifest.ttl

```diff
@@ -1,961 +1,1430 @@
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/turtle"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/n-triples"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/html"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/ld+json"
+                ])
+            ]);
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "CONSTRUCT query appropriate content type (expect one of: RDF/XML, Turtle, N-Triples, RDFa, JSON-LD))" .
+
+<#query_content_type_describe> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "query=DESCRIBE%20%3Chttp%3A%2F%2Fexample.org%2F%3E&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf"
+        ];
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:headers ([
+              a ht:RequestHeader;
+              ht:fieldName "Content-Type";
+              ht:fieldValue "application/rdf+xml, text/turtle, application/n-triples, text/html, application/ld+json";
+              ht:headerElements ([
+                  a ht:HeaderElement;
+                  ht:elementName "application/rdf+xml"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/turtle"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/n-triples"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "text/html"
+                ] [
+                  a ht:HeaderElement;
+                  ht:elementName "application/ld+json"
+                ])
+            ]);
+          ht:statusCodeValue "2XX",
+            "3XX"
+        ]
+      ])
+  ];
+  mf:name "DESCRIBE query appropriate content type (expect one of: RDF/XML, Turtle, N-Triples, RDFa, JSON-LD)" .
+
+<#query_content_type_select> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "query=SELECT%20%281%20AS%20%3Fvalue%29%20%7B%7D&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf"
```

Comment:
Original test did not specify a dataset, and specified query via direct-POST request body. This request is missing `Content-Type: application/sparql-query` header.